README.md

```diff
@@ -35,6 +35,11 @@ you want to search, the extension will response the related search results insta
 ## Contribution
 
 ```bash
+# Installing jsonnet first if you havn't installed it yet, we employ jsonnet to generate manifest.json
+sudo apt update
+sudo apt install snapd
+sudo snap install jsonnet
```

Comment:
I prefer a platform-agnostic `jsonnet` installation link here rather than a Linux install instruction. 

For example, we can say:

`jsonnet` is required before getting started. To install `jsonnet`, please check `jsonnet`'s [README](https://github.com/google/jsonnet#packages).